test/run/test/exec-args.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect, test } from 'vitest'
+import { execa } from 'execa'
+import { runVitest } from '../../test-utils'
+
+test.each([
+  { pool: 'forks', execArgv: ['--hash-seed=1', '--random-seed=1', '--no-opt'] },
+  { pool: 'threads', execArgv: ['--inspect-brk'] },
+  { pool: 'vmThreads', execArgv: ['--inspect-brk'] },
+] as const)('should pass execArgv to { pool: $pool } ', async ({ pool, execArgv }) => {
+  const fileToTest = `exec-args-fixtures/${pool}.test.ts`
+
+  const vitest = await runVitest({
+    include: [fileToTest],
+    pool,
+    poolOptions: {
+      [pool]: {
+        execArgv,
+      },
+    },
+  })
+
+  expect(vitest.stdout).toContain(`✓ ${fileToTest}`)
+})
+
+test('should not pass execArgv to workers when not specified in the config', async () => {
+  const { stdout } = await execa('npx', ['vitest', '--run', '--root', 'no-exec-args-fixtures', '--config', 'vitest.config.ts'],
+    { env: { NODE_OPTIONS: '--inspect-brk' } })
```

Comment:
This does not work as expected, `--inspect-brk` is not passed. I will try and find a way to populate `process.execArgv` when using `npx`.

Reply:
As there was no way of doing it here, I modified the loaded `vitest.config.ts` to add the flag that we do not want to be passed